Source/Android/jni/Cheats/GeckoCheat.cpp

```diff
@@ -58,7 +59,7 @@ Java_org_dolphinemu_dolphinemu_features_cheats_model_GeckoCheat_getCreator(JNIEn
 JNIEXPORT jstring JNICALL
 Java_org_dolphinemu_dolphinemu_features_cheats_model_GeckoCheat_getNotes(JNIEnv* env, jobject obj)
 {
-  return ToJString(env, JoinStrings(GetPointer(env, obj)->notes, "\n"));
+  return ToJString(env, fmt::format("{}", fmt::join(GetPointer(env, obj)->notes, "\n")));
```

Comment:
Can do `fmt::to_string(fmt::join(GetPointer(env, obj)->notes, "\n"))` if you want something shorter...